### PR TITLE
WIP: Unit/User permissions list view

### PR DESCRIPTION
**Pull request by otiala**

### CODE REVIEW

File: respa_admin/static_src/styles/base.scss [hunk 1]
```diff
@@ -29,4 +29,5 @@ $icon-font-path: "~bootstrap-sass/assets/fonts/bootstrap/";
 @import "./page-office";
 @import "./user-matrix";
 @import "./page-resource";
+@import "./page-user-management-list.scss";
```
Comment (jussih) [inline respa_admin/static_src/styles/base.scss]: I'm interested - why is this import listed with file extension while the others are not?

File: respa_admin/templates/respa_admin/resources/_unit_user_list.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                    </div>
+                </form>
+            </div>
+        </div>
+
+        <div class='container-fluid'>
+            {% for unit in units %}
+            <span class="sub-header">{{ unit.name }}</span>
+
+            {% for authorization in unit.authorizations.all %}
```
Comment (jussih) [inline respa_admin/templates/respa_admin/resources/_unit_user_list.html]: It's a bit hard to follow nested loops without indentation

File: respa_admin/urls.py [hunk 3]
```diff
@@ -4,7 +4,9 @@
 from . import views
 from .views.resources import (
     ResourceListView,
-    SaveResourceView
+    SaveResourceView,
```
Comment (jussih) [inline respa_admin/urls.py]: These in alphabetical order :)

File: respa_admin/templates/respa_admin/resources/_unit_user_list.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                        <a href="">{% trans 'Edit permissions' %}</a>
+                    </h6>
+                    </div>
+                </div>
+            </div>
+
+            {% endwith %}
+            {% endfor %}
+            {% endfor %}
+            {% include "respa_admin/common/_pagination.html" %}
```
Comment (jussih) [inline respa_admin/templates/respa_admin/resources/_unit_user_list.html]: I'm pretty sure pagination won't work with this template include. The included snippet uses variables such as `page_obj`, `selected_resource_type`, `selected_resource_unit` and `order_by` which do not exist in this template context. Or the pagination support in DRF might include `page_obj`, but not not the others at least.

File: respa_admin/templates/respa_admin/resources/_unit_user_list.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                            <g id="a-edit-outline-64" stroke="#000000" stroke-width="2">
+                                <path d="M5.25,6.53333333 C5.25,2.92506667 8.188125,0 11.8125,0 C15.436875,0 18.375,2.92506667 18.375,6.53333333 C18.375,10.1416 15.436875,14 11.8125,14 C8.188125,14 5.25,10.1416 5.25,6.53333333 Z" id="Shape"></path>
+                                <path d="M17.5,18.8591375 C15.9526723,18.5699062 14.1147601,18.375 11.9733691,18.375 C6.84324097,18.375 3.45661956,19.4967938 1.63344386,20.3418688 C0.635049078,20.80435 0,21.838075 0,22.9791188 L0,28 L11.9733691,28" id="Shape"></path>
+                                <polygon id="Shape" points="19.3641304 27.5434783 16.625 28 17.0815217 25.2608696 24.8423913 17.5 27.125 19.7826087">
+                                </polygon>
+                            </g>
+                        </g>
+                    </svg>
+                    <h6>
+                        <a href="">{% trans 'Edit permissions' %}</a>
```
Comment (jussih) [inline respa_admin/templates/respa_admin/resources/_unit_user_list.html]: Hide the `Edit permissions` link if the user is request.user - a user should not be able to edit their own permissions.

This also needs to be enforced in the permission edit functionality in the next issue.

File: respa_admin/static_src/styles/base.scss (hunk 1 above)
Comment (otiala) [reply respa_admin/static_src/styles/base.scss]: No particular reason, I was unaware of the magic handling the imports. Will change.

**Comment by jussih** [comment]
Change the PR to point to develop
